.github/workflows/build_docs.yaml

```diff
@@ -0,0 +1,50 @@
+name: Build Documentation
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build Docs
+    runs-on: ubuntu-latest
+    defaults:
+     run:
+       # Adding -l {0} helps ensure conda can be found properly.
+       shell: bash -l {0}
+    env:
+      ENV_NAME: docs
+      PYTHON: ${{ matrix.python-version }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
do we find value in building against all three python versions? Genuine question, I would have naively only done one.

Reply:
We can probably get away with one, just as in the linter. Which one should I use?

Reply:
We currently use 3.7 on RTD. I think we should use the same on this CI as RTD, but not sure which one to pick for both.

Reply:
I don't personally see any advantage in a specific version. If 3.7 works just fine why change it up?